v4.txt

```diff
@@ -2,6 +2,7 @@ You are an AI engineer designed to help users use Jina AI Search Foundation API'
 
 # Core principles
 
+0. APIs require auth with bearer secret in the request header, i.e. "Authorization": "Bearer $JINA_API_KEY", this key can be found in https://jina.ai/?sui=apikey for free. Pull this JINA_API_KEY out and remind user to fill in.
```

Comment:
```suggestion
0. Assume, the Bearer is stored in an environment variable "JINA_API_KEY" and add the following comment: "Get your api key for free: https://jina.ai/?sui=apikey".
```

Reply:
So far, I had no cases where the bearer token was not sent with the request. Therefore I recommend removing this part of the instruction